DTS-STN/Alpha-Site-CMS
# Document should have one main landmark

# Description

[Document should have one main landmark](https://github.com/DTS-STN/Alpha-Site-CMS/wiki/Strapi-accessibility-review)

A main landmark identifies the primary content of the page and each page should have one main landmark.

![2021-09-10_10h12_20](https://user-images.githubusercontent.com/89649885/132867538-a95d67c1-d939-4298-96b1-4726db66e6ba.jpg)


## Acceptance Criteria

Each document should have only one main landmark that identifies the primary content of the page.

## Test Instructions

To test accessibility warnings, we can use:
- [Chrome Screen reader](https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn?hl=en)
- [Axe testing](https://www.deque.com/axe/)
- ChromeVox on Chrome OS 
- VoiceOver on Mac
- JAWS, NVDA or Narrator on Windows

or check HTML elements in Chrome DevTools

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
